splunk-apm/README.md

```diff
@@ -1,22 +1,24 @@
-# Splunk distribution of OpenTelemetry Lambda
+# Splunk APM layer for AWS lambda
 
-Splunk OTeL lambda is a downstream distribution of opentelemetry-lambda. Supported OpenTelemetry Lambda layers are preconfigured to use Splunk as the tracing backend (direct ingest or SmartAgent). Users can enhance their existing Lambda functions by adding Splunk-managed layer directly. Layer ARNs [are published here](https://github.com/signalfx/lambda-layer-versions/).
+Splunk APM layer for AWS lambda contains various components enabling seamless integration of lambdas with metrics and traces. The components come preconfigured to use Splunk as the backend (direct ingest or SmartAgent). Users can enhance their existing Lambda functions by adding Splunk-managed layer directly. Layer ARNs [are published here](./splunk-apm.md).
```

Comment:
Instead of SmartAgent perhaps this should mention splunk opentelemetry connector

Reply:
Hmmm, why are we hiding OTel here?

Reply:
We don't :D It's just that first layer contained only OTeL-based tracing wrapper while now it's also SFx (Splunk) metrics extension. So Splunk APM layer in general is more that OTeL.